packages/svelte/src/internal/client/dom/elements/class.js

```diff
@@ -14,7 +14,47 @@ export function class_name_effect(dom, value) {
 }
 
 /**
- * @param {Element} dom
+ * @param {SVGElement} dom
+ * @param {() => string} value
+ * @returns {void}
+ */
+export function svg_class_name_effect(dom, value) {
+	render_effect(() => {
+		svg_class_name(dom, value());
+	});
+}
+
+/**
+ * @param {SVGElement} dom
+ * @param {string} value
+ * @returns {void}
+ */
+export function svg_class_name(dom, value) {
```

Comment:
I think it makes sense to code-golf this: add a boolean which is true by default, and when `true` it uses the regular dom element variant, else the "always attribute" variant.